homeassistant/components/wemo.py

```diff
@@ -91,6 +92,7 @@ def discovery_dispatch(service, discovery_info):
         if device is None:
             device = pywemo.discovery.device_from_description(url, None)
 
-        discovery_info = (device.name, device.model_name, url, device.mac)
+        discovery_info = (device.name, device.model_name, url, device.mac,
```

Comment:
I remember messing up when I added mac. Doesn't this also require a matching change in `netdisco`